apps/studio/data/tables/keys.ts

```diff
@@ -1,6 +1,10 @@
 export const tableKeys = {
-  list: (projectRef: string | undefined, schema: string | undefined) =>
-    ['projects', projectRef, 'tables', { schema }] as const,
+  list: (
+    projectRef: string | undefined,
+    schema: string | undefined,
+    includeColumns?: boolean | undefined
+  ) =>
+    ['projects', projectRef, 'tables', { schema }, { includeColumns: !!includeColumns }] as const,
```

Comment:
Any reason you want another object in the key? I've always thought of the final object as kind of the "options" object.
```suggestion
    ['projects', projectRef, 'tables', { schema, includeColumns: !!includeColumns }] as const,
```